selectrum.el

```diff
@@ -754,8 +735,7 @@ candidate and return the candidate as displayed."
              selectrum--refined-candidates)
     (if notfull
         selectrum--refined-candidates
-      (cl-loop for cand in selectrum--refined-candidates
-               collect (selectrum--get-full cand)))))
+      (mapcar #'selectrum--get-full selectrum--refined-candidates))))
```

Comment:
Why?

Reply:
I stumbled over this when looking through the changes but its actually not worth discussing for me.